<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierDialog</class>
 <widget class="QWidget" name="CashierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cierre de caja</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: black;
	background-color: rgb(237, 233, 230);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Montos:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI Variable</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Información de la caja actual.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="accept_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(0, 153, 73);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(0, 181, 84);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(0, 125, 58);     /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(232, 54, 18);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(253, 59, 20);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(208, 48, 16);    /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cash_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">	background-color: white;
	color: black;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>Efectivo: 20.000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sinpe_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	background-color: white;
	color: black;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>Sinpe: 5.000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="card_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">	background-color: white;
	color: black;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>Tarjeta: 3.000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="total_label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">	background-color: white;
	color: black;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>Total: 3000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Desgloce del monto total de la caja en su correpondiente metodo de pago.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gastos:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QScrollArea" name="scrollArea_expenses">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: white;
	color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ExpensesAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1162</width>
        <height>118</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Recibos:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QScrollArea" name="scrollArea_receipts">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: white;
	color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ReceiptsAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1162</width>
        <height>198</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
